modules/rhcos-about.adoc

```diff
@@ -9,7 +9,7 @@
 
 {op-system} is supported only as a component of {product-title} {product-version} for all {product-title} machines. {op-system} is the only supported operating system for {product-title} control plane, or master, machines. While {op-system} is the default operating system for all cluster machines, you can create compute machines, which are also known as worker machines, that use {op-system-base} as their operating system. There are two general ways {op-system} is deployed in {product-title} {product-version}:
 
-* If you install your cluster on infrastructure that the cluster provisions, {op-system} images are downloaded to the target platform during installation, and suitable Ignition config files, which control the {op-system} configuration, are used to deploy the machines.
+* If you install your cluster on infrastructure that the installation program provisions, {op-system} images are downloaded to the target platform during installation, and suitable Ignition config files, which control the {op-system} configuration, are used to deploy the machines.
```

Comment:
Looks like installation program is the right term here. My only comment would be if possible to break this sentence up into two or three. It's a very hard sentence to read and I worry about translation or non-native English speakers having a hard time understanding this sentence.

Suggestion (or something like this):

If you install your cluster on an infrastructure that the installation program provisions, {op-systems} images are downloaded to the target platform during installation. Suitable Ignition config files, which control the {op-system} configuration, are also downloaded and used to deploy the machines.